### DISCUSSION TITLE
SNUtils Compare, how to save

### DISCUSSION THREAD

**Discussion by Sauron-Dev** [arnoudkooi/SN-Utils#320] [Q&A] [answered]
Hello, I wonder how you can save changes that you made in the SNUtils Compare screen? I don't see a save button or something like this. Maybe I am using the function the wrong way?

Thank you very much for the explanation :)

**Comment by arnoudkooi** [comment, marked as answer]
No, it is just a compare, save is not implemented...

https://twitter.com/sn_utils/status/1487116211104239617

**Comment by Sauron-Dev** [comment]
Oh thank you, I should pay attention to your twitter feed! I was searching for the save because you have implemented already the arrow in the middle bar to move code from left to right, thats why I was thinking there should also be a save, haha :) Thank you for your great efforts (and ultra fast feedback).
